src/org/usfirst/frc/team694/robot/util/LimeLight.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Lime Light Client:
+ * Issues? Feel free to contact me as Sam Belliveau on Slack.
+ * the purpose of this project is to directly communicate with the Lime Light.
+ * This extends the included NetworkTableClient by taking table names from the
+ * Lime Light Documents. Using this will let you spend less time trouble
+ * shooting with Network Tables and spend more time making usable.
+ *
+ * Lime Light Docs: http://docs.limelightvision.io/en/latest/networktables_api.html#
+ */
+
+package org.usfirst.frc.team694.util;
+
+import org.usfirst.frc.team694.util.NetworkTableClient;
+
+public enum LED_MODE {
+  PIPELINE(0), FORCE_OFF(1), FORCE_BLINK(2), FORCE_ON(3);
+
+  private int codeValue;
+
+  LED_MODE(int value) { this.codeValue = value; }
+  public int getCodeValue() { return codeValue; }
+};
+
+public enum CAM_MODE {
+  VISION(0), DRIVER(1);
+
+  private int codeValue;
+
+  CAM_MODE(int value) { this.codeValue = value; }
+  public int getCodeValue() { return codeValue; }
+};
+
+public enum STREAM {
+  STANDARD(0), PIP_MAIN(1), PIP_SECONDARY(2);
```

Comment:
What does PIP stand for? Might be worth a comment here

Reply:
Picture in picture. Search for `stream` in http://docs.limelightvision.io/en/latest/networktables_api.html